src/ducks/settings/Configuration.jsx

```diff
@@ -226,10 +225,6 @@ export class Configuration extends React.Component {
   }
 }
 
-Configuration.defaultProps = {
-  hideReimbursements: false
-}
-
```

Comment:
Nit: This commit could be squashed with the one introducing the change. This would mkae the history cleaner, and is fairly easy to do with gitup.